lms/static/js/fixtures/instructor_dashboard/membership.html

```diff
@@ -96,7 +96,7 @@ <h3 class="hd hd-3">Course Team Management</h3>
 
     <div class="auth-list-container" data-rolename="Community TA" data-display-name="Discussion Community TA"
          data-info-text="
-          Community TAs are members of the community who help course teams moderate discussions. They can see posts by all learners, and can edit or delete posts, clear flags, close or re-open threads, and endorse responses. Their posts are marked as 'Community TA'. Only enrolled users can be added as Community TAs."
+          Community TAs are members of the community who help course teams moderate discussions. They can see posts by learners in their assigned cohort/enrollment track group, and can edit or delete posts, clear flags, close or re-open threads, and endorse responses. Their posts are marked as 'Community TA'. Only enrolled users can be added as Community TAs."
```

Comment:
Change "cohort/enrollment track group" to "cohort or enrollment track" in all locations